src/UCTNode.h

```diff
@@ -65,7 +65,7 @@ class UCTNode {
     int get_visits() const;
     float get_score() const;
     void set_score(float score);
-    float get_eval(int tomove) const;
+    float get_eval(int tomove, bool no_VL) const;
```

Comment:
No caps. Also you should probably default this to = false and simplify this patch a lot.

Reply:
So you agree with @roy7 about using an option, it should just default? I may have to rebase anyway to connect this with #1388 which was pulled in the meantime...

Reply:
I don't feel strongly either way, but I think a separate function is a tiny bit cleaner. I'm writing that as we speak, so no need tor rebase.